lldb/source/Target/Target.cpp

```diff
@@ -1452,7 +1452,7 @@ static void LoadTypeSummariesForModule(ModuleSP module_sp) {
     return;
 
   Log *log = GetLog(LLDBLog::DataFormatters);
-  const char *module_name = module_sp->GetObjectName().GetCString();
+  const char *module_name = module_sp->GetFileSpec().GetFilename().GetCString();
```

Comment:
Micro-optimization: since this is only used in LLDB_LOGF, we can save the C string conversion by using LLDB_LOG instead and use a StringRef.